anitya/lib/versions/rpm.py

```diff
@@ -48,10 +48,7 @@
 
     warnings.warn("Failed to import 'rpm', emulating RPM label comparisons")
 
-    try:
-        from itertools import zip_longest
-    except ImportError:
-        from itertools import izip_longest as zip_longest
+    from itertools import zip_longest
```

Comment:
This was probably needed by old version of python, I'm glad this could be removed.

Reply:
Great!